packages/@lwc/engine-server/README.md

```diff
@@ -13,14 +13,25 @@ This package supports the following APIs.
 
 This function renders a string-representation of a serialized component tree, given a tag name
 and an LWC constructor. The output format itself is aligned with the [current leading
-proposal][explainer], but is subject to change.
+proposal][explainer], but is subject to change. The result is a pair {html,styles} with
+html being the markup, while styles is undefined for native Shadow DOM.
+
+It also supports synthetic-shadow by generating the proper Light DOM markup. In this case, the result
+returns an array of inline styles to be injected in the parent, typically the head of the document.
 
 ```js
 import { renderComponent } from 'lwc';
 import LightningHello from 'lightning/hello';
 
 const componentProps = {};
-const serialized = renderComponent('lightning-hello', LightningHello, componentProps);
+
+// Render native Shadow DOM
+const { html } = renderComponent('lightning-hello', LightningHello, componentProps);
+
+// Render synthetic Shadow DOM
+const { html, styles } = renderComponent('lightning-hello', LightningHello, componentProps, {
```

Comment:
can we use a different API instead?

Reply:
We could but when we did the first code review, this had been the preferred option. Whatever works with me, just let me know.

Reply:
I proposed reusing the same API for consistency purposes. The idea to keep the same API and pass a flag is that LWC uses the same API for both native and synthetic shadow.